app/views/support/addresses/edit.html.erb

```diff
@@ -0,0 +1,35 @@
+<%- title = t('page_titles.support.addresses.edit') %>
+<%- content_for :title, title_with_error_prefix(title, @school.errors.any?) %>
+<%- content_for :before_content do %>
+<%- end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= title %>
+    </h1>
+
+    <% if @school.computacenter_change == 'none' %>
+      <p class="govuk-inset-text">
```

Comment:
I don't think we need `govuk-inset-text` as well as the tag.

Reply:
This is now just the tag and the inset has been removed